client/CServerHandler.h

```diff
@@ -86,6 +89,8 @@ class CServerHandler : public IServerAPI, public LobbyInfo
 
 	std::vector<std::string> myNames;
 
+	std::shared_ptr<HighScoreCalculation> calc;
```

Comment:
More descriptive name?
```suggestion
	std::shared_ptr<HighScoreCalculation> highScoreCalc;
```